ghost/i18n/locales/da/portal.json

```diff
@@ -1,148 +1,148 @@
+    "{{memberEmail}} will no longer receive this newsletter.": "{{memberEmail}} vil ikke længere modtage dette nyhedsbrev.",
     "{{trialDays}} days free": "{{trialDays}} dage gratis",
     "A login link has been sent to your inbox. If it doesn't arrive in 3 minutes, be sure to check your spam folder.": "Der er blevet sendt en link til din email, du kan bruge til at logge ind med. Hvis det ikke kommer frem indenfor 3 minutter, så tjek din spam mappe.",
     "Account": "Konto",
     "Account settings": "Kontoindstillinger",
     "After a free trial ends, you will be charged the regular price for the tier you've chosen. You can always cancel before then.": "Når din gratis prøveperiode udløber, vil du blive opkrævet den normale pris for det abonnement du har valgt. Du kan selvfølgelig altid annullere dit abonnement inden.",
     "Already a member?": "Er du allerede medlem?",
-    "An unexpected error occured. Please try again or <a>contact support</a> if the error persists.": "",
+    "An unexpected error occured. Please try again or <a>contact support</a> if the error persists.": "Der opstod en uventet fejl. Prøv venligst igen eller <a>kontakt support</a> hvis fejlen fortsætter.",
     "Back": "Tilbage",
     "Back to Log in": "Tilbage til log ind",
-    "Billing info": "",
-    "Black Friday": "",
-    "Cancel anytime.": "",
+    "Billing info": "Faktureringsoplysninger",
+    "Black Friday": "Black Friday",
+    "Cancel anytime.": "Annuller når som helst.",
     "Cancel subscription": "Annuller abonnement",
     "Cancellation reason": "Årsag til annullering",
-    "Change": "",
-    "Check spam & promotions folders": "",
-    "Check with your mail provider": "",
-    "Choose": "",
+    "Change": "Skift",
+    "Check spam & promotions folders": "Tjek spam og kampagner mapper",
+    "Check with your mail provider": "Tjek med din e-mailudbyder",
+    "Choose": "Vælg",
     "Choose a different plan": "Vælg et andet abonnement",
     "Choose your newsletters": "Hvilke nyhedsbreve ønsker du",
-    "Click here to retry": "",
+    "Click here to retry": "Klik her for at prøve igen",
     "Close": "Luk",
     "Comments": "Kommentarer",
-    "Complimentary": "",
+    "Complimentary": "Gratis",
     "Confirm": "Bekræft",
-    "Confirm cancellation": "",
-    "Confirm subscription": "",
-    "Contact support": "",
+    "Confirm cancellation": "Bekræft annullering",
+    "Confirm subscription": "Bekræft abonnement",
+    "Contact support": "Kontakt support",
     "Continue": "Fortsæt",
-    "Continue subscription": "",
-    "Could not sign in. Login link expired.": "",
-    "Could not update email! Invalid link.": "",
-    "Create a new contact": "",
-    "Current plan": "",
+    "Continue subscription": "Fortsæt tilmelding",
+    "Could not sign in. Login link expired.": "Kunne ikke log ind. Log ind link udløbet.",
```

Comment:
I would change this on to: **Kunne ikke logge find. Dit link til at logge ind med er udløbet.**

Reply:
The English original suggests a short form, hence my proposal. I would agree with you if the original contained full sentences